misk-redis/src/testFixtures/kotlin/misk/redis/testing/FakeRedis.kt

```diff
@@ -42,6 +50,9 @@ class FakeRedis @Inject constructor(
   private val hKeyValueStore =
     ConcurrentHashMap<String, Value<ConcurrentHashMap<String, ByteString>>>()
 
+  private val sortedSetKeyValueStore =
+    ConcurrentHashMap<String, Value<SortedMap<ByteString, Double>>>()
```

Comment:
`SortedMap` sorts by key, not value. So it won't provide you the same ordering as a Redis sorted set which sorts by "value", i.e. score. You might be better off using a map for now to keep the implementation simple.

Reply:
You are right, this is a mistake on my part. I changed this. Added a comment here.